5-infrastructure/README.md

```diff
@@ -6,44 +6,61 @@ All infrastructure components will be created using the base network created dur
 
 ## Components
 
-- 3 GKE Clusters
-    - Cluster1 in Primary Region
-    - CLuster2 in Secondary Region
-    - MCI Cluster is Primary Region
-- Bastion Host VM in the Secondary Region
-- 2 Postgres CLoudSQL instances in primary and secondary region respectively
-- Private Services Access for CloudSQL Instances
-- 1 Secret to store CloudSQL Admin Password
-- 4 KMS Keyrings and Keys
-    - 2 KMS Keyrings and Keys for GKE, one in each region
-    - 2 KMS Keyrings and Keys for CloudSQL, one in each region
-- Service Account for KMS to own/manage the Keyrings and Keys
-- Service Account for Bastion VM with roles to install Anthos Service Mesh
-- 4 Log Sinks, one in each project
-- Log Sink Destination Storage Bucket that Log Sinks write logs to
+- **3 GKE Clusters**
+  - Cluster1 in the primary region (us-east1)
+  - Cluster2 in the secondary region (us-west1)
+  - MCI Cluster in the primary region (us-east1)
+- **Bastion Host VM** in the secondary region (us-west1)
+- **2 Postgres CLoudSQL instances** in the primary and secondary regions, respectively
+- **Secret** to store the CloudSQL Admin Password
+- **4 KMS Keyrings and Keys**
+  - 2 KMS Keyrings and Keys for GKE, one in each region
+  - 2 KMS Keyrings and Keys for CloudSQL, one in each region
+- **Service Account for KMS** to own/manage the Keyrings and Keys
+- **Service Account for the Bastion Host VM** with roles to install Anthos Service Mesh
+- **4 Log Sinks**, one in each project
+- **Log Sink Destination Storage Bucket** that Log Sinks write logs to
+- **Cloud Armor Policy**
+- **External IP** for accessing the application externally
 
 ## Prerequisites
 
-1. 0-bootstrap executed successfully.
-1. 1-org executed successfully.
-1. 2-environments executed successfully.
-1. 3-networks executed successfully.
-1. 4-projects executed successfully.
+1. [0-bootstrap](https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/0-bootstrap/README.md) executed successfully.
+1. [1-org](https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/1-org/README.md) executed successfully.
+1. [2-environments](https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/2-environments/README.md) executed successfully.
+1. [3-networks](https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/3-networks/README.md) executed successfully.
+1. [4-projects](../app-foundation/4-projects/README.md) executed successfully.
+
+**Troubleshooting:**
+If your user does not have access to run the commands above and you are in the organization admins group, you can append `--impersonate-service-account="boa-terraform-<z>-sa@prj-bu1-<z>-boa-sec-<xxxx>.iam.gserviceaccount.com"` for dev/npd/prd envs or `--impersonate-service-account="cicd-build-sa@prj-bu1-s-app-cicd-<xxxx>.iam.gserviceaccount.com"` for shared env to run the command as the terraform service account.
```

Comment:
What are the comments above?

Reply:
changed to `**Troubleshooting:**
If your user does not have access to run the terraform modules locally and you are in the organization admins group, you can append `--impersonate-service-account=org-terraform@<SEED_PROJECT_ID>.iam.gserviceaccount.com` to run terraform modules as the service  account.`